terraform/variables/prod-us.tfvars

```diff
@@ -86,66 +116,96 @@ ingestors = {
       # ta-ta is reserved for integration testing with NCI and MITRE's staging
       # instances
       ta-ta = {
-        intake_worker_count                    = 1
-        aggregate_worker_count                 = 1
+        min_intake_worker_count                = 1
+        max_intake_worker_count                = 3
+        min_aggregate_worker_count             = 1
+        max_aggregate_worker_count             = 3
         peer_share_processor_manifest_base_url = "test-en-analytics.cancer.gov"
         portal_server_manifest_base_url        = "manifest.int.enpa-pha.io"
       }
       us-ct = {
-        intake_worker_count    = 3
-        aggregate_worker_count = 3
+        min_intake_worker_count    = 1
+        max_intake_worker_count    = 6
+        min_aggregate_worker_count = 1
+        max_aggregate_worker_count = 3
       }
       us-md = {
-        intake_worker_count    = 3
-        aggregate_worker_count = 3
+        min_intake_worker_count    = 1
+        max_intake_worker_count    = 6
+        min_aggregate_worker_count = 1
+        max_aggregate_worker_count = 3
       }
       us-va = {
-        intake_worker_count    = 3
-        aggregate_worker_count = 3
+        min_intake_worker_count    = 1
+        max_intake_worker_count    = 6
+        min_aggregate_worker_count = 1
+        max_aggregate_worker_count = 3
       }
       us-wa = {
-        intake_worker_count    = 3
-        aggregate_worker_count = 3
+        min_intake_worker_count    = 1
+        max_intake_worker_count    = 6
+        min_aggregate_worker_count = 1
+        max_aggregate_worker_count = 3
       }
       us-ca = {
-        intake_worker_count    = 15
-        aggregate_worker_count = 3
+        min_intake_worker_count    = 1
+        max_intake_worker_count    = 10
```

Comment:
nit: I'm almost certain this is intentional, but this shrinks the maximum number of intake workers for us-ca from 15 to 10. I'm OK with it if you are (I suspect you know better than me what numbers are reasonable here), just want to make sure it's intentional.

Reply:
(and same observation for the Apple us-ca deployment)

Reply:
Yes: my thinking here is that we have been overprovisioned in CA for a while. Fewer than 25 or 15 workers for Apple or Google respectively should be enough to stay on top of the backlog. If it's not, we can easily crank up the `max_intake_worker_count`.